custom_components/tuya_local/devices/nexsmart_air_2.yaml

```diff
@@ -0,0 +1,237 @@
+name: AIR 2 Camera (Battery)

```

Comment:
Top level names should be generic, without any branding. Also avoid punctuation characters in device (and entity) names, as they get used as ids in places.